zerver/lib/error_notify.py

```diff
@@ -64,7 +64,8 @@ def email_browser_error(report: Dict[str, Any]) -> None:
 User agent: {user_agent}
 href: {href}
 Server path: {server_path}
-Deployed version: {version}
+Deployed server version: {server_version}
+Web version: {web_version}
```

Comment:
Cool! This is great because with deferred restarts, these don't always match! 

(I think previously you had to kinda visually skim it from the blueslip log if one cared; super ughy).

Reply:
Yup, catching those was the intent.